themes/hugoplate/layouts/partials/essentials/head.html

```diff
@@ -29,7 +31,7 @@
 
 <!-- google analytics -->
 {{ if and site.Config.Services.GoogleAnalytics.ID (ne site.Config.Services.GoogleAnalytics.ID "G-G6825VQMZB") }}
-<!-- {{ template "_internal/google_analytics.html" . }} -->
+  <!-- {{ template "_internal/google_analytics.html" . }} -->
 {{ end }}
 
```

Comment:
we don't use this one right, since we use gtm
I think I'll go and remove things from the themes from now on whenever I stumble over them

Reply:
Yes, feel free to do so.